.docs/dev-guide/build-reference.md

```diff
@@ -4,7 +4,56 @@ How to build libStorage
 
 ---
 
-## Build Requirements
+## Basic Builds
+The following one-line command is the quickest, simplest, and most
+deterministic approach to building libStorage:
+
+```bash
+$ git clone https://github.com/codedellemc/libstorage && make -C libstorage
+```
```

Comment:
I think the only thing I see missing is an explanation that if Docker is present, it will be used by default (even if you just do `make` and not `make docker-build`. If Docker is not present, it builds locally, just like `make build`.